src/app-gocardless/services/gocardless-service.js

```diff
@@ -429,7 +430,7 @@ export const goCardlessService = {
    * @throws {ServiceError}
    * @returns {Promise<import('../gocardless.types.js').GetTransactionsResponse>}
    */
-  getTransactions: async ({ accountId, startDate, endDate }) => {
+  getTransactions: async ({ institutionId, accountId, startDate, endDate }) => {
     const response = await client.getTransactions({
       accountId,
       dateFrom: startDate,
```

Comment:
we need to somehow remove `dataTo` parameter when calling `getTransactions`. I am not sure if we can remove it for all providers or we need to find a way to remove it for Norwegian SE alone? 🤔 
See more details in the PR comments

Reply:
@fstybel @MatissJanis Is there a strong reason for having a set end date for this request?  I feel like we should be able to just stop setting one at all.

Reply:
Don't remember there being a strong reason. I'd be fine removing it, but in a separate PR please (so we can roll back easily if something breaks).

Reply:
Great, I'll split it out now.

Reply:
Moved to #241.